app/router.js

```diff
@@ -98,6 +98,7 @@ Router.map(function() {
   this.route('search', { path: '/zoeken', }, function() {
     this.route('cases', { path: '/dossiers', });
     this.route('agenda-items', { path: '/agendapunten', });
+    this.route('newsletter-infos', { path: '/kort-bestek', });
```

Comment:
@MikiDi 'newsletters' ipv 'newsletter-infos' ? Dan trek ik dat wat gelijk naar hoe andere routes noemen
 `this.route('newsletters', { path: '/kort-bestek', });`